### PR TITLE
sample for checkout borrow license

### PR DESCRIPTION
**Pull request by rajyaramazon**
*Issue #, if available:*

E2E example for checkout borrow license

*Description of changes:*

sample for checkout borrow license

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: managed-entitlements/src/managed_entitlements_checkout_borrow_sample.py [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+    return response['PublicKey']
+
+def main(command_line=None):
+    print("Start of the sample model for checkout borrow license")
+    # Initially, you should create an asymmetric key that License Manager uses to
+    # sign borrow license data. The algorithms are based on JWT standard and all
+    # major languages has public libraries to verify signatures. For this sample,
+    # we are using PyJwt
+    SignKey=create_cmk()
```
Comment (bukka17) [inline managed-entitlements/src/managed_entitlements_checkout_borrow_sample.py:109]: We need to be clear that this needs to be created in the region where you are doing checkoutborrow. Key and checkoutborrow should be in the same region

File: managed-entitlements/src/managed_entitlements_checkout_borrow_sample.py [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+    # and are indicating to License Manager that it should allow license borrowing
+    # by specifying BorrowConfiguration.
+    License=create_license(SignKey=SignKey)
+    LicenseArn=License["LicenseArn"]
+    print(f"License Created: {LicenseArn}")
+
+    # Borrowing a license. For real scenarios, it should be also accounted the
+    # distribution and potential leaks. Use borrowing specifying and validating
+    # NodeId if distribution is not safe.
+    signed_token=checkout_borrow_license(LicenseArn)["SignedToken"]
```
Comment (bukka17) [inline managed-entitlements/src/managed_entitlements_checkout_borrow_sample.py:128]: Add comment here on the region of the key.

File: managed-entitlements/src/managed_entitlements_checkout_borrow_sample.py [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+            "Begin": "2021-04-12T00:00:00Z"
+        },
+        LicenseMetadata=[{
+            "Name":"ProductName",
+            "Value":"My awesome product"
+        }],
+        ClientToken=str(uuid.uuid4())
+    )
+
+def checkout_borrow_license(LicenseArn):
```
Comment (bukka17) [inline managed-entitlements/src/managed_entitlements_checkout_borrow_sample.py:57]: Lets add some explanation on when we use the API.
Also some notes on if the token expires or not for checked out borrow tokens.

File: managed-entitlements/src/managed_entitlements_checkout_borrow_sample.py (hunk 1 above)
Comment (rajyaramazon) [reply managed-entitlements/src/managed_entitlements_checkout_borrow_sample.py:110]: Addressed in new revision

File: managed-entitlements/src/managed_entitlements_checkout_borrow_sample.py (hunk 2 above)
Comment (rajyaramazon) [reply managed-entitlements/src/managed_entitlements_checkout_borrow_sample.py:130]: Addressed in new revision

File: managed-entitlements/src/managed_entitlements_checkout_borrow_sample.py (hunk 3 above)
Comment (rajyaramazon) [reply managed-entitlements/src/managed_entitlements_checkout_borrow_sample.py:57]: Added in read me

**Review by rajyaramazon — COMMENTED** [review]
sample for checkout borrow license

**Pull request merged**